app/views/courses/show.html.erb

```diff
@@ -11,11 +11,20 @@
         <% @errors.each do |id, messages| %>
           <% messages.each do |message| %>
             <li data-error-message="<%= message %>">
-              <a href="<%= enrichment_error_url(
-                provider_code: @provider.provider_code,
-                course: @course,
-                field: id,
-              ) %>"><%= message %></a>
+              <% if message == 'You must say whether you have additional degree requirements' %>
```

Comment:
It gets shown on the validations banner at the top of the description page when you try and publish a course. You'd need to checkout the branch that my PR adds it in on the backend for it to work though https://github.com/DFE-Digital/teacher-training-api/pull/1992

Reply:
![image](https://user-images.githubusercontent.com/42515961/123003321-f39d3780-d3aa-11eb-8c79-ba8f65d7bbf7.png)

Reply:
I'm not too sure what I am seeing `additional` vs `minimum`

![image](https://user-images.githubusercontent.com/470137/123068470-d51a5900-d409-11eb-8795-2e27c87ffaab.png)

![additional_vs_minimum](https://user-images.githubusercontent.com/470137/123068785-1e6aa880-d40a-11eb-9fed-34ce706a0013.gif)

Reply:
@davidgisbey ah – can you change this message to "Enter degree requirements" please, to match the blue text further down the page? This covers both the minimum degree requirements and the subject requirements pages.

In the scenario where a provider has answered the minimum degree requirements questions but NOT the degree subject questions (which should be super rare as they're presented one after the other) it could either still link to the first (already answered) question, or jump to the second question – either is probably fine.

Reply:
Have done!